packages/app/README.md

```diff
@@ -36,7 +36,7 @@ npm install @h5web/app
 ```
 
 ```tsx
-import '@h5web/app/dist/styles.css';
```

Comment:
The old path is still supported, but the documentation should now assume that consumers use modern bundlers that support `"exports"`.